frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.js

```diff
@@ -0,0 +1,292 @@
+            onChange={onChangeBoundingBox}
+            filterOption={(input, option) =>
+              option.children.toLowerCase().indexOf(input.toLowerCase()) >= 0
+            }
+          >
+            {userBoundingBoxes.map(userBB => (
+              <Select.Option key={userBB.id} value={userBB.id}>
+                {renderUserBoundingBox(userBB)}
+              </Select.Option>
+            ))}
+          </Select>
+        </div>
+        <br />
+      </React.Fragment>
+    ) : null;
+
+  const hasUnselectedOptions =
+    selectedColorLayerName == null || (isBoundingBoxConfigurable && selectedBoundingBox == null);
+
+  return (
+    <Modal
+      title={`Start ${capitalizeWords(jobName)}`}
+      onCancel={handleClose}
+      visible
+      width={700}
+      footer={null}
+    >
+      {description}
+      <br />
+      <div style={{ textAlign: "center" }}>
+        <img
+          //TODO: get an image
+          src="/assets/images/nuclei_inferral_example.jpg"
+          alt={`${jobName} example`}
+          style={{ width: 400, height: "auto", borderRadius: 3 }}
+        />
+      </div>
+      <br />
+      <ColorLayerSelection />
+      <BoundingBoxSelection />
+      <div style={{ textAlign: "center" }}>
+        <Button type="primary" disabled={hasUnselectedOptions} onClick={startJob}>
+          Start {capitalizeWords(jobName)}
+        </Button>
+      </div>
+    </Modal>
+  );
+}
+
+function _NucleiInferralModal({ dataset, handleClose, userBoundingBoxes }: Props) {
+  return (
+    <StartingJobModal
+      dataset={dataset}
+      userBoundingBoxes={userBoundingBoxes}
+      handleClose={handleClose}
+      jobName="nuclei inferral"
+      jobApiCall={colorLayerName =>
+        startNucleiInferralJob(dataset.owningOrganization, dataset.name, colorLayerName)
+      }
+      description={
+        <>
+          <p>
+            Start a job that automatically detects nuclei for this dataset. This job creates a copy
+            of this dataset once it has finished. The new dataset will contain the detected nuclei
+            as a segmentation layer.
+          </p>
+          <p>
+            <b>
+              Note that this feature is still experimental. Nuclei detection currently works best
+              with EM data and a resolution of approximately 200{ThinSpace}nm per voxel. The
+              inferral process will automatically use the magnification that matches that resolution
+              best.
+            </b>
+          </p>
+        </>
+      }
+    />
+  );
+}
+
+function _NeuronInferralModal({ dataset, handleClose, userBoundingBoxes }: Props) {
+  return (
+    <StartingJobModal
+      dataset={dataset}
+      handleClose={handleClose}
+      userBoundingBoxes={userBoundingBoxes}
+      jobName="neuron inferral"
+      isBoundingBoxConfigurable
+      jobApiCall={async (colorLayerName, boundingBox) => {
+        if (!boundingBox) {
+          return Promise.resolve();
```

Comment:
```suggestion
          return;
```

should also work afaik, since the function is already async and thus always returns a promise.

Reply:
Sadly, this does work for eslint.
> Async arrow function expected no return value. eslint(consistent-return)

 I think disabling this rule here is not work the effort. Do you agree?

Reply:
Ok, then let's keep it as is :+1: